### ISSUE TITLE
[Potential bug] Once PR title is modified kicking the tests again gets the old PR name

### ISSUE THREAD

**Issue by bastienboutonnet** [seferov/pr-lint-action#20]
I'm not entirely sure if this is expected behaviour, but I tested the action with a PR that does not match the regex I had put and I get a fail (that is expected)

After editing the name of the PR in github and re runing the checks it looks like the action pulls the unmodified title (which was `Add pr lint action` but has been changed to `Feature: Add pr lint action`

```
title: Add pr lint action
##[error]Please fix your PR title to match ^Feature:|Fix:|Refactor:
##[error]Docker run failed with exit code 1
```

**Comment by seferov** [comment]
Hello @bastienboutonnet 

This is not expected but known issue. Unfortunately Github doesn't trigger any event after change of PR title. To overcome this, either pull request must be updated via push for example or close/open PR. There is a feature request (auto-close invalid PRs) https://github.com/seferov/pr-lint-action/issues/19 may cover this issue.

**Comment by bastienboutonnet** [comment]
Thanks for getting back to me on this.
Yeah I figured it was probably a limitation from what was available to the action by github. Feel free to close this issue as there doesn't seem to be something we can do about it. 

I'll keep an eye on when/if #19 will get implemented and see if the workflow is nice for the user but it doesn't sound like a bad one.

**Comment by seferov** [comment]
@bastienboutonnet this should fix it to some extend. When title is edited, it re-runs the job but adds new job alongside with failing status

<img width="480" alt="image" src="https://user-images.githubusercontent.com/1233843/69478271-a5583900-0e01-11ea-8cd2-e34ff17304bc.png">

**Comment by bastienboutonnet** [comment]
Ah nice, this may not be too shabby actually. I'm gonna give it a go! Thanks @seferov
